docs/reference/cluster.asciidoc

```diff
@@ -9,13 +9,15 @@
 Most cluster level APIs allow to specify which nodes to execute on (for
 example, getting the node stats for a node). Nodes can be identified in
 the APIs either using their internal node id, the node name, address,
-custom attributes, or just the `_local` node receiving the request. For
+custom attributes, or just the `_local`, `_master` or `_coordinating_only` nodes receiving the request. For
```

Comment:
This doesn't make sense - `_local` refers to the node receiving the request, but `_master` and `_coordinating_only` refer to all master and coordinating nodes. I think it might be better to expand this section of the docs with something like a bulleted list, since this sentence is becoming unwieldy.

Reply:
Hi @DaveCTurner, I just find this part a little bit confusing when we say `"nodes to execute on"` or `"node receiving the request"`. If I understand well, all the `_local`, `_master`, `_coordinating_only` or node id, custom attributes etc. are just filters right ? When I do a `GET /_nodes/_local` it returns to me the node info of the local node, if I do a `GET /_nodes/_master`, it returns to me the node info of master node ? same for other api like nodes stats, nodes hot_threads etc. So if the local node is the master, I get the same result for the above 2 requests right ? What we would explain here is not "the node to which request was forwarded" ?